### ISSUE TITLE
Bot responses failing to take after creating a common template

### ISSUE THREAD

**Issue by JonathanFingold** [microsoft/BotFramework-Composer#7888]
## Describe the bug

Replace an existing response with a template. Press Enter. End up with the **Send a response** property pane not agreeing with the authoring canvas.

## Version

Version:  2.0.0-nightly.246170.8fa93
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser

- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## To Reproduce

1. Create a bot with a skill.
1. Make sure the skill has a trigger with a **Send a response** action in it.
1. Go to the **Bot responses** view. Fight with the editor for a while, give up, switch to show code, and add this template to the common responses for the skill.
    ```lg
    # DescribeWeather(weather)
    - It is "${dialog.weather}" in ${dialog.city} in ${dialog.country} and the temperature is ${dialog.fahrenheit}&deg;F or ${dialog.celsius}&deg;C. Have a nice day...
    ````
1. Go back to the **Create** view and the **Send a response** action in the skill.
1. Remove the existing response, then add the following in its place:
    ```lg
    ${DescribeWeather(dialog.weather)}
    ```
1. Error: Notice that the authoring canvas still shows the old template value.
1. Error: Select a different action, then reselect the **Send a response** action. Notice the value you just entered, repeatedly, has been clobbered, again.

## Expected behavior

It works, or it displays some error explaining why it has not took. It does not pretend to sort of work.

## Screenshots

![image](https://user-images.githubusercontent.com/35047199/119204248-d4c52180-ba49-11eb-9c01-bacf70e74cd6.png)

**Comment by emgrol** [comment]
We tried a few scenarios. I have a stand alone root bot (created from the Empty bot template) and don't have any issues. I suspect this may be a skill issue.

**Comment by hatpick** [comment]
@JonathanFingold can you attach a bot project (compressed) to repro this issue?

**Comment by hatpick** [comment]
@JonathanFingold I believe this was resolved by PR #7842. that was merged in on Friday. Can you check the latest nightly to see if your issue is resolved.
